gatsby_fp/src/components/fieldComponents/intellContComponents/SubHeading.js

```diff
@@ -0,0 +1,104 @@
+import React from "react"
+const shortid = require("shortid")
+
+export default function IntellCont({ intellContList, publicationType }) {
+
+    return (
+    <>
+      <div className="publicationtitle">{publicationType}</div>
+        <ol>
+          {intellContList.filter(element => element.contype === publicationType).map(element => (
+            <li
+              key={shortid.generate()}
+              dangerouslySetInnerHTML={{
+                __html: ` 
+                 ${
+                    element.status
+                      ? `${element.status}`
+                      : ``
+                  }
+                  ${
+                    element.contype
+                      ? `${element.contype} -`
+                      : ``
+                  }
+                  ${
+                    element.intellcont_auth.lname
+                      ? `${element.intellcont_auth.lname},`
+                      : ``
+                  }
+                  ${
+                    element.intellcont_auth.fname
+                      ? `${element.intellcont_auth.fname}.`
+                      : ``
+                  }
+                  (2020).
+                  ${
+                    element.title
+                      ? `${element.title}.`
```

Comment:
Unfortunately, this results in the period being outside of the closing quote which is usually considered improper. The closing quote is baked into the `title` value though, so I'd rather not mess around with trying to position the period properly via string manipulation. We will need to discuss.